lua/lib/populate.lua

```diff
@@ -23,6 +25,7 @@ local function dir_new(cwd, name)
     match_name = path_to_matching_str(name),
     match_path = path_to_matching_str(absolute_path),
     open = false,
+    children_initial = has_children,
```

Comment:
i'd prefer keeping the has_children nomenclature, i had trouble understanding `children_initial` until i saw this.

Reply:
So the reason I decided to switch from `has_children` to `children_initial`, is because this variable is only used until a folder is expanded for the first time, i.e. when it gets all of its items populated. Before being populated, the entries are set to `{}`, no matter what the real contents of the folder are, so this is a temporary or "initial" variable that simple gets set to true if there is at least 1 item within the folder.

I am not against changing the name back, but I think distinguishing that it is an "initial" or temporary variable is a good idea. Let me know if it should be changed back to `has_children`

Reply:
I see, i'd still like has_children better though :)